rs/ledger_suite/icrc1/ledger/tests/tests.rs

```diff
@@ -182,30 +186,6 @@ fn test_upgrade() {
     ic_ledger_suite_state_machine_tests::test_upgrade(ledger_wasm(), encode_init_args)
 }
 
-#[test]
-fn test_install_mainnet_ledger_then_upgrade_then_downgrade() {
```

Comment:
Shouldn't these tests be only commented out (I assume they should be reenabled once all migration steps are rolled out)?